src/cr/cube/matrix/assembler.py

```diff
@@ -149,19 +149,19 @@ def columns_margin(self):
         # --- TODO: Should column_margin only be defined when it's 1D? This would
         # --- require changes to exporter to use the bases to give a
         # --- "column_margin_range"
-        if not self._measures.columns_margin.is_defined:
+        if not self._measures.columns_weighted_counts.is_defined:
```

Comment:
@gergness The naming convention is that marginals are singular and (body) measures are plural. The `rows_` and `columns_` prefix helps differentiate, but I think this name needs to be singular.

In this case, I think the name you want is `columns_weighted_base`, meaning "these are the base for each column". The fact that they are computed from counts does not directly describe their role.

Reply:
I now see that the scale measures can't treat the margin as a true 1D strand, which is what you were hinting at. So it's true we can't simplify by treating them as true strands.

However, this isn't really why I think of them as counts.
> The fact that they are computed from counts does not directly describe their role.

It's true that counts and bases have different roles, but in the situations that a 1D version of these "bases for a margins" exists, we know there is no array, so the counts and bases are equal. To me, the base's main role is that it's used as the denominator for a proportion calculation.

A part of my consolidation though has been to use the 2D `row_weighted_bases` as the denominator for `row_proportions`, so we don't actually use them as a denominator very often. Instead, the 1D form is actually used as the numerator of the `margin_table_proportions`. 

All of that to say, I think it's a little weird to use something called a "base" as a numerator, but I can see how the "count" name is also confusing, especially if we want to keep the name singular to indicate it's on a margin.